src/wrapper.cpp

```diff
@@ -45,8 +45,10 @@ TokenException* GetOrCreateLastException() {
   static base::NoDestructor<base::ThreadLocalPointer<TokenException>>
       last_exception;
   TokenException* token_exception = last_exception.get()->Get();
-  if (!token_exception)
-    last_exception.get()->Set(new TokenException(""));
+  if (!token_exception) {
+    token_exception = new TokenException("");
+    last_exception.get()->Set(token_exception);
+  }
```

Comment:
Can we have an automatic test that covers this code path?

Reply:
evq confirms this was tested downstream by bat-native-confirmations; see also https://github.com/brave-intl/challenge-bypass-ristretto-ffi/issues/40 and https://github.com/brave-intl/challenge-bypass-ristretto-ffi/issues/41.